#!/usr/bin/python3
def safe_function(fct, *args):
    import sys
    try:
        x = fct(*args)
    except Exception as err:
        x = None
        print("Exception:", err, file=sys.stderr)
    return x
